RecoLocalCalo/HcalRecProducers/src/HBHERecHitProducerGPU.cc

```diff
@@ -239,6 +215,10 @@ void HBHERecHitProducerGPU::acquire(edm::Event const& event,
   setup.get<HcalSiPMCharacteristicsRcd>().get(sipmCharacteristicsHandle);
   auto const& sipmCharacteristicsProduct = sipmCharacteristicsHandle->getProduct(ctx.stream());
 
+  edm::ESHandle<HcalMahiPulseOffsetsGPU> pulseOffsetsHandle;
+  setup.get<HcalMahiPulseOffsetsGPURecord>().get(pulseOffsetsHandle);
+  auto const& pulseOffsetsProduct = pulseOffsetsHandle->getProduct(ctx.stream());
```

Comment:
Just to note that eventually these need to be migrated to ESGetToken (but probably a separate PR is better)
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#Getting_data_from_EventSetup_wit

Reply:
I guess we need to do it once we move to CMSSW_11_2_X ?

Reply:
> I guess we need to do it once we move to CMSSW_11_2_X ?

The "new code must use ESGetToken" is  rather a policy than technical requirement (so "need" is before making a PR to CMSSW `master`). Technically the migration can be done at any time, the ESGetToken API has been there already for over a year.

Reply:
OK - I was wondering if it is technically enforced in 11.2.x.
If it isn't "before making the PR for master" (or rather "while the PR for master is being reviewed") seems like a good moment :-)

Reply:
It's not enforced (yet) for EDModules because of ̃~5500 existing calls that need to be migrated first :)